src/js/read.js

```diff
@@ -69,7 +69,7 @@ function createMarkupForOneDate(date, news) {
             abstract: newsEntity.snippet,
             title: newsEntity.title,
             updated: newsEntity.newsdate,
-            nytdsection: newsEntity.sectionname,
+            nytdsection: newsEntity.sectionName,
```

Comment:
тут треба повернути назву sectionname
```suggestion
            nytdsection: newsEntity.sectionname,
```